src/System.Buffers/src/System/Buffers/DefaultArrayPool.cs

```diff
@@ -23,23 +24,25 @@ internal DefaultArrayPool(int maxLength, int arraysPerBucket)
             // Our bucketing algorithm has a minimum length of 16
             if (maxLength < MinimiumArraySize)
                 maxLength = MinimiumArraySize;
-            
+
+            int id = GetHashCode();
```

Comment:
Weird, I thought that was part of a commit before...sorry about that, fixed again